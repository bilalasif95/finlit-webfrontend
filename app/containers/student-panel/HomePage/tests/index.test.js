import React from 'react';
import { render } from 'react-testing-library';
import { IntlProvider } from 'react-intl';
import { BrowserRouter } from 'react-router-dom';
import { HomePage } from '../index';

describe('<HomePage />', () => {
  it('should render and match the snapshot', () => {
    const {
      container: { firstChild },
    } = render(
      <IntlProvider locale="en">
        <BrowserRouter>
          <HomePage />
        </BrowserRouter>
      </IntlProvider>,
    );

    expect(firstChild).toMatchSnapshot();
  });
});
