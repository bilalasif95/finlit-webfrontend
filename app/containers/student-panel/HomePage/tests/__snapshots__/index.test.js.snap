// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage /> should render and match the snapshot 1`] = `
<div
  class="main_page"
>
  <div
    class="Wrapper-sc-170e4vb-0 eETvPP"
  >
    <div
      class="container-xl"
    >
      <div
        class="row"
      >
        <div
          class="col-sm-12 col-md-1 col-lg-2"
        />
        <div
          class="col-sm-12 col-md-10 col-lg-8"
        >
          <div
            class="main_content"
          >
            <h4>
              <span>
                How Can We Help?
              </span>
            </h4>
            <div
              class="obj_cont"
            >
              <div
                class="circle"
              >
                circle
              </div>
              <img
                alt="dots"
                class="dots"
                src="IMAGE_MOCK"
              />
              <div
                class="search_bar"
              >
                <div
                  class="input-group"
                >
                  <input
                    class="form-control"
                    placeholder="Search for help on specific topics"
                    type="text"
                  />
                  <div
                    class="input-group-append"
                  >
                    <button
                      class="btn btn-secondary"
                    >
                      <span>
                        Search
                      </span>
                    </button>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="col-sm-12 col-md-1 col-lg-2"
        />
      </div>
    </div>
  </div>
  <div
    class="Wrapper-ly91ui-0 WpvMC"
    id="services"
  >
    <div
      class="container-xl"
    >
      <div
        class="row"
      >
        <div
          class="col-12 col-sm-6 col-md-3 col-lg-3"
        >
          <a
            class="single_item"
            href="/accredited_education_list"
          >
            <img
              alt="Accredited Education"
              src="IMAGE_MOCK"
            />
            <h5>
              <span>
                Accredited Education
              </span>
            </h5>
            <p>
              <span>
                Lorem ipsum vitea eros dictum, gravida massa id, pellentesque est.
              </span>
            </p>
          </a>
        </div>
        <div
          class="col-12 col-sm-6 col-md-3 col-lg-3"
        >
          <a
            class="single_item"
            href="/live_webinars_list"
          >
            <img
              alt="Live Webinars"
              src="IMAGE_MOCK"
            />
            <h5>
              <span>
                Live Webinars
              </span>
            </h5>
            <p>
              <span>
                Lorem ipsum vitea eros dictum, gravida massa id, pellentesque est.
              </span>
            </p>
          </a>
        </div>
        <div
          class="col-12 col-sm-6 col-md-3 col-lg-3"
        >
          <a
            class="single_item"
            href="/login"
          >
            <img
              alt="Goal Based Education"
              src="IMAGE_MOCK"
            />
            <h5>
              <span>
                Goal Based Education
              </span>
            </h5>
            <p>
              <span>
                Lorem ipsum vitea eros dictum, gravida massa id, pellentesque est.
              </span>
            </p>
          </a>
        </div>
        <div
          class="col-12 col-sm-6 col-md-3 col-lg-3"
        >
          <a
            class="single_item"
            href="/features"
          >
            <img
              alt="Free Gifts &amp; Rewards"
              src="IMAGE_MOCK"
            />
            <h5>
              <span>
                Free Gifts & Rewards
              </span>
            </h5>
            <p>
              <span>
                Lorem ipsum vitea eros dictum, gravida massa id, pellentesque est.
              </span>
            </p>
          </a>
        </div>
      </div>
    </div>
  </div>
  <div
    class="Wrapper-naedyx-0 hTcsrt"
    id="services"
  >
    <div
      class="container-xl"
    >
      <div
        class="row"
      >
        <div
          class="col-lg-12"
        >
          <div
            class="header"
          >
            <h4>
              <span>
                Financial Literacy
              </span>
            </h4>
            <a
              href="/"
            >
              <span>
                View all
              </span>
            </a>
          </div>
        </div>
      </div>
      <div
        class="row"
      >
        <div
          class="col-12 col-sm-6 col-md-3 col-lg-3"
        >
          <div
            class="single_item"
          >
            <img
              alt="Accredited Education"
              src="IMAGE_MOCK"
            />
            <h5>
              <span>
                Accredited Education
              </span>
            </h5>
            <p>
              <span>
                Lorem ipsum vitea eros dictum, gravida massa id, pellentesque est.
              </span>
            </p>
            <a
              class="read_more"
              href="/"
            >
              <span>
                Read more
              </span>
            </a>
          </div>
        </div>
        <div
          class="col-12 col-sm-6 col-md-3 col-lg-3"
        >
          <div
            class="single_item"
          >
            <img
              alt="Budgeting"
              src="IMAGE_MOCK"
            />
            <h5>
              <span>
                Budgeting
              </span>
            </h5>
            <p>
              <span>
                Lorem ipsum vitea eros dictum, gravida massa id, pellentesque est.
              </span>
            </p>
            <a
              class="read_more"
              href="/"
            >
              <span>
                Read more
              </span>
            </a>
          </div>
        </div>
        <div
          class="col-12 col-sm-6 col-md-3 col-lg-3"
        >
          <div
            class="single_item"
          >
            <img
              alt="Credit Profile - What is credit?"
              src="IMAGE_MOCK"
            />
            <h5>
              <span>
                Credit Profile - What is credit?
              </span>
            </h5>
            <p>
              <span>
                Lorem ipsum vitea eros dictum, gravida massa id, pellentesque est.
              </span>
            </p>
            <a
              class="read_more"
              href="/"
            >
              <span>
                Read more
              </span>
            </a>
          </div>
        </div>
        <div
          class="col-12 col-sm-6 col-md-3 col-lg-3"
        >
          <div
            class="single_item"
          >
            <img
              alt="Loans & Debt"
              src="IMAGE_MOCK"
            />
            <h5>
              <span>
                Loans & Debt
              </span>
            </h5>
            <p>
              <span>
                Lorem ipsum vitea eros dictum, gravida massa id, pellentesque est.
              </span>
            </p>
            <a
              class="read_more"
              href="/"
            >
              <span>
                Read more
              </span>
            </a>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="Wrapper-hzz68y-0 fAJGsz"
    id="services"
  >
    <div
      class="container-xl"
    >
      <div
        class="row"
      >
        <div
          class="col-lg-12"
        >
          <div
            class="header"
          >
            <h4>
              <span>
                Entrepreneurship
              </span>
            </h4>
            <a
              href="/"
            >
              <span>
                View all
              </span>
            </a>
          </div>
        </div>
      </div>
      <div
        class="row"
      >
        <div
          class="col-12 col-sm-6 col-md-3 col-lg-3"
        >
          <div
            class="single_item"
          >
            <img
              alt="Accredited Education"
              src="IMAGE_MOCK"
            />
            <h5>
              <span>
                Accredited Education
              </span>
            </h5>
            <p>
              <span>
                Lorem ipsum vitea eros dictum, gravida massa id, pellentesque est.
              </span>
            </p>
            <a
              class="read_more"
              href="/"
            >
              <span>
                Read more
              </span>
            </a>
          </div>
        </div>
        <div
          class="col-12 col-sm-6 col-md-3 col-lg-3"
        >
          <div
            class="single_item"
          >
            <img
              alt="Live Webinars"
              src="IMAGE_MOCK"
            />
            <h5>
              <span>
                Live Webinars
              </span>
            </h5>
            <p>
              <span>
                Lorem ipsum vitea eros dictum, gravida massa id, pellentesque est.
              </span>
            </p>
            <a
              class="read_more"
              href="/"
            >
              <span>
                Read more
              </span>
            </a>
          </div>
        </div>
        <div
          class="col-12 col-sm-6 col-md-3 col-lg-3"
        >
          <div
            class="single_item"
          >
            <img
              alt="Goal Based Education"
              src="IMAGE_MOCK"
            />
            <h5>
              <span>
                Goal Based Education
              </span>
            </h5>
            <p>
              <span>
                Lorem ipsum vitea eros dictum, gravida massa id, pellentesque est.
              </span>
            </p>
            <a
              class="read_more"
              href="/"
            >
              <span>
                Read more
              </span>
            </a>
          </div>
        </div>
        <div
          class="col-12 col-sm-6 col-md-3 col-lg-3"
        >
          <div
            class="single_item"
          >
            <img
              alt="Free Gifts &amp; Rewards"
              src="IMAGE_MOCK"
            />
            <h5>
              <span>
                Free Gifts & Rewards
              </span>
            </h5>
            <p>
              <span>
                Lorem ipsum vitea eros dictum, gravida massa id, pellentesque est.
              </span>
            </p>
            <a
              class="read_more"
              href="/"
            >
              <span>
                Read more
              </span>
            </a>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="Wrapper-sc-1cidyo7-0 jxLDOs"
    id="services"
  >
    <div
      class="container-xl"
    >
      <div
        class="row"
      >
        <div
          class="col-lg-12"
        >
          <div
            class="header"
          >
            <h4>
              <span>
                Articles
              </span>
            </h4>
          </div>
        </div>
      </div>
      <div
        class="row"
      >
        <div
          class="col-12 col-sm-12 col-md-6 col-lg-6"
        >
          <div
            class="single_item"
          >
            <div
              class="left"
            >
              <img
                alt="Icon"
                src="IMAGE_MOCK"
              />
              <h5>
                <span>
                  Interactive Tools
                </span>
              </h5>
              <p>
                <span>
                  Lorem ipsum vitea eros dictum, gravida massa id, pellentesque est.
                </span>
              </p>
              <a
                class="read_more"
                href="/article_details"
              >
                <span>
                  Read more
                </span>
              </a>
            </div>
            <div
              class="right"
            >
              <img
                alt="Interactive Tools"
                src="IMAGE_MOCK"
              />
            </div>
          </div>
        </div>
        <div
          class="col-12 col-sm-12 col-md-6 col-lg-6"
        >
          <div
            class="single_item"
          >
            <div
              class="left"
            >
              <img
                alt="icon"
                src="IMAGE_MOCK"
              />
              <h5>
                <span>
                  Live Webinars
                </span>
              </h5>
              <p>
                <span>
                  Lorem ipsum vitea eros dictum, gravida massa id, pellentesque est.
                </span>
              </p>
              <a
                class="read_more"
                href="/article_details"
              >
                <span>
                  Read more
                </span>
              </a>
            </div>
            <div
              class="right"
            >
              <img
                alt="Simulations"
                src="IMAGE_MOCK"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="Wrapper-sc-1t9wbrl-0 cnOZBM"
    id="services"
  >
    <div
      class="container-xl"
    >
      <div
        class="row"
      >
        <div
          class="col-lg-12"
        >
          <div
            class="header"
          >
            <h4>
              Recent Events
            </h4>
            <a
              href="/"
            >
              View all
            </a>
          </div>
        </div>
      </div>
      <div
        class="row"
      >
        <div
          class="col-12 col-sm-6 col-md-6 col-lg-3"
        >
          <div
            class="single_item"
          >
            <div
              class="event_img"
            >
              <img
                alt="Event"
                src="IMAGE_MOCK"
              />
              <div
                class="title"
              >
                <p>
                  Bootcamp
                </p>
              </div>
            </div>
            <div
              class="desc"
            >
              <h5>
                Online Fitness Bootcamp mit Sanda - Mo lunch
              </h5>
              <p>
                Mon, Aug 16, 2021 6:00 PM PKT
              </p>
              <div
                class="posted_by"
              >
                <div
                  class="posted_profile"
                >
                  <img
                    alt="Publisher"
                    src="IMAGE_MOCK"
                  />
                </div>
                <p
                  class="posted_title"
                >
                  <span>
                    Sandra Zangeri
                  </span>
                </p>
              </div>
            </div>
          </div>
        </div>
        <div
          class="col-12 col-sm-6 col-md-6 col-lg-3"
        >
          <div
            class="single_item"
          >
            <div
              class="event_img"
            >
              <img
                alt="Event"
                src="IMAGE_MOCK"
              />
              <div
                class="title"
              >
                <p>
                  Bootcamp
                </p>
              </div>
            </div>
            <div
              class="desc"
            >
              <h5>
                Online Fitness Bootcamp mit Sanda - Mo lunch
              </h5>
              <p>
                Mon, Aug 16, 2021 6:00 PM PKT
              </p>
              <div
                class="posted_by"
              >
                <div
                  class="posted_profile"
                >
                  <img
                    alt="Publisher"
                    src="IMAGE_MOCK"
                  />
                </div>
                <p
                  class="posted_title"
                >
                  <span>
                    Sandra Zangeri
                  </span>
                </p>
              </div>
            </div>
          </div>
        </div>
        <div
          class="col-12 col-sm-6 col-md-6 col-lg-3"
        >
          <div
            class="single_item"
          >
            <div
              class="event_img"
            >
              <img
                alt="Event"
                src="IMAGE_MOCK"
              />
              <div
                class="title"
              >
                <p>
                  Bootcamp
                </p>
              </div>
            </div>
            <div
              class="desc"
            >
              <h5>
                Online Fitness Bootcamp mit Sanda - Mo lunch
              </h5>
              <p>
                Mon, Aug 16, 2021 6:00 PM PKT
              </p>
              <div
                class="posted_by"
              >
                <div
                  class="posted_profile"
                >
                  <img
                    alt="Publisher"
                    src="IMAGE_MOCK"
                  />
                </div>
                <p
                  class="posted_title"
                >
                  <span>
                    Sandra Zangeri
                  </span>
                </p>
              </div>
            </div>
          </div>
        </div>
        <div
          class="col-12 col-sm-6 col-md-6 col-lg-3"
        >
          <div
            class="single_item"
          >
            <div
              class="event_img"
            >
              <img
                alt="Event"
                src="IMAGE_MOCK"
              />
              <div
                class="title"
              >
                <p>
                  Bootcamp
                </p>
              </div>
            </div>
            <div
              class="desc"
            >
              <h5>
                Online Fitness Bootcamp mit Sanda - Mo lunch
              </h5>
              <p>
                Mon, Aug 16, 2021 6:00 PM PKT
              </p>
              <div
                class="register"
              >
                <a
                  href="/accredited_education_list"
                >
                  <span>
                    Register for FREE
                  </span>
                </a>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
